src/win/thread.c

```diff
@@ -144,6 +144,11 @@ int uv_thread_create_ex(uv_thread_t* tid,
     ResumeThread(thread);
   }
 
+  /* Detach the thread if UV_THREAD_DETACHED is set */
+  if (params->flags & UV_THREAD_DETACHED) {
```

Comment:
Don't we want to do this in the else block?

Reply:
WDYM?